programs/obfuscator/Obfuscator.cpp

```diff
@@ -186,7 +187,7 @@ static UInt64 transform(UInt64 x, UInt64 seed)
     if (x == 2 || x == 3)
         return x ^ (seed & 1);
 
-    size_t num_leading_zeros = __builtin_clzll(x);
+    size_t num_leading_zeros = std::countl_zero(x);
```

Comment:
Why don't we use only `src/Common/BitHelpers.h`?
That way we avoid replacing all the occurrences in the future.
We can also cast signed integers there.

Reply:
I would say that a "limitation" of the std lib does not really justify introduction of a wrapper + asking everyone to use the wrapper (besides the mental overhead of juggling two functions doing the same job). I put "limitation" in quotes because disallowing signed integers is a valid option from stdlib POV  given that until very recently (C++20), the standard did not specify if signed integers are in one's or two's complement 😁 

Will remove the last users of getLeading/TrailingZeroBits() (file "Common/BitHelpers.h") and then these two functions will also go. C++20 + some deliberate casting make them obsolete.

Reply:
> In some cases, we continue to use buildin_c(l|t)z(), (e.g. in
src/Common/BitHelpers.h) because the std:: versions only accept
unsigned inputs (and they also check that) and the casting would be
ugly.

Then I would go with casting.
If accepting signed integers is something STL functions don't like, maybe it's good to have it ugly so you know what you're doing. 
Just sharing my opinion I'm fine with leaving it as is, but if we have STL functions let's try to use them as much as possible. 😄